README.md

```diff
@@ -1,70 +1,66 @@
-# Getting Started with Create React App
+# Math-Magician App
```

Comment:
- You need to check your readme for the correct project title and description. Kindly change this title to your correct project name, as well as the description, this will make your documentation more professional.